airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MicroMeterRegistryFactory.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.metrics.lib;
+
+import static io.airbyte.metrics.lib.MetricClientFactory.DATADOG_METRIC_CLIENT;
+
+import io.airbyte.config.Configs;
+import io.airbyte.config.EnvConfigs;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.statsd.StatsdConfig;
+import io.micrometer.statsd.StatsdMeterRegistry;
+
+public class MicroMeterRegistryFactory {
```

Comment:
why not integrate this with the existing MetricClientFactory?

Reply:
good point. fixed.